apps/demos/Demos/Pagination/Overview/jQuery/index.js

```diff
@@ -0,0 +1,100 @@
+$(() => {
+  const pagination = $('#pagination')
+    .dxPagination({
+      showInfo: true,
+      lightModeEnabled: false,
+      pagesNavigatorVisible: true,
+      showNavigationButtons: true,
+      allowedPageSizes: [4, 6],
+      itemCount: employees.length,
+      pageIndex: 1,
+      pageSize: 4,
+      onOptionChanged: (evt) => {
+        if (evt.name === 'pageSize') {
+          pagination.option('pageSize', evt.value);
+          renderEmployeeGallery();
+        }
+
+        if (evt.name === 'pageIndex') {
+          pagination.option('pageIndex', evt.value);
+          renderEmployeeGallery();
+        }
+      },
+    })
+    .dxPagination('instance');
+
+  const createEmployeeImg = (employee) => {
```

Comment:
I'd extract methods to a separate file like employeeGallery.js

Reply:
fixed